sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/BasicRelationship.java

```diff
@@ -118,13 +118,13 @@ public BasicRelationship setName(String name) {
     }
 
     /**
-     * Sets the additional properties defined in the model. This field will contain any properties of the relationship that are not already defined by the other strong types of this class.
+     * Adds an additional property to this model. This field will contain any properties of the relationship that are not already defined by the other strong types of this class.
      * @param key The key of the additional property to be added to the relationship.
      * @param value The value of the additional property to be added to the relationship.
      * @return The BasicRelationship object itself.
      */
     @JsonAnySetter
-    public BasicRelationship setCustomProperties(String key, Object value) {
```

Comment:
A setter would take the full map, not a single key value pair. So I just renamed this to more accurately describe what this method does